src/components/menu/menu.tsx

```diff
@@ -64,13 +67,15 @@ export function Menu() {
   useHotkeys('shift+alt+p', () => open('presets'));
   useHotkeys('shift+h', () => open('shortcuts'));
   useHotkeys('shift+s', () => open('shareLink'), { enabled: !noSelected });
+  useHotkeys('shift+t', () => open('sleepTimer'));
 
   useCloseListener(closeAll);
 
   const variants = mix(fade(), slideY());
 
   return (
     <>
+      <Divider />
```

Comment:
Is this left by accident? If not, what purpose does it serve? It creates a line between the two toolbar buttons that isn't aesthetically pleasing.

Reply:
Yup, accidental ! Sorry and thanks for noticing !